src/main/java/io/supertokens/inmemorydb/queries/MfaQueries.java

```diff
@@ -33,7 +34,7 @@ public static String getQueryToCreateUserFactorsTable(Start start) {
         return "CREATE TABLE IF NOT EXISTS " + Config.getConfig(start).getMfaUserFactorsTable() + " ("
                 + "app_id VARCHAR(64) DEFAULT 'public',"
                 + "tenant_id VARCHAR(64) DEFAULT 'public',"
-                + "user_id VARCHAR(255) NOT NULL,"
+                + "user_id VARCHAR(128) NOT NULL,"
                 + "factor_id VARCHAR(255) NOT NULL,"
```

Comment:
factor id here is inconsistent with psql..

Reply:
done. set to 16.